app/models/solution/mentor_discussion.rb

```diff
@@ -38,4 +38,12 @@ def viewable_by?(user)
     # TODO: Admins should also be allowed to view
     [mentor, student].include?(user)
   end
+
+  def mark_as_nothing_to_do(user)
```

Comment:
@iHiD This okay?

Reply:
Let's name it similar to the method below

```suggestion
  def does_not_require_mentor_action!
  end
```